discord/http.py

```diff
@@ -1149,6 +1154,7 @@ def start_forum_thread(
         auto_archive_duration: threads.ThreadArchiveDuration,
         rate_limit_per_user: int,
         invitable: bool = True,
+        applied_tags: SnowflakeList | None,
```

Comment:
Shouldn't this have a default value?
```suggestion
        applied_tags: SnowflakeList | None = None,
```